airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx

```diff
@@ -19,7 +19,7 @@ const FetchMessage = styled.div`
 
 const LoadingSchema: React.FC = () => (
   <SpinnerBlock>
-    <Spinner />
+    <ProgressBar runTime={120} />
```

Comment:
Can we make this 4min when running discover? 2min seems good for check connection.